pre/pipelines.conf

```diff
@@ -0,0 +1,5 @@
+[connection:MQTTConnection]
```

Comment:
_Commented on notebook `pre/pipelines.conf` Cell 1 Line 2_
```python
[connection:MQTTConnection]
broker=mosquitto:1883
```
The ASAB config format supports $ env vars. Can we have those for the broker, username + password so we don't have to mount a config file into the container just to set this?

_Respond and view the context [here](https://gitnotebooks.com/bitswan-space/BitSwan/pull/13#comment-1143)._